### ISSUE TITLE
Add actions logging to debugger options

### ISSUE THREAD

**Issue by Leooo** [emberjs/ember.js#16866]
It would be very useful to have a `debugging` option to log all `actions` called, together with their arguments. Hopefully this would be something not to hard to add to current Ember infrastructure.

Looks like it's one of the nice things in Redux, to be able to log actions (and state changes, which probably would be harder in ember-data).

**Comment by pixelhandler** [comment]
@Leooo Yeah that does sound nice, currently we have these tools: https://guides.emberjs.com/release/configuring-ember/debugging/ and https://guides.emberjs.com/release/ember-inspector/ which does track promises but no flag to enable logging for every action that is fired. If an action is not handled an error will be thrown so that does help.

Perhaps create an **Issue** in the RFCs repo, see https://github.com/emberjs/ember.js/blob/master/CONTRIBUTING.md#requesting-a-feature

This is small enough not to need a PR on the RFC repo, just an open issue for the feature request.
